recipes/dsm2dtm/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "dsm2dtm" %}
+{% set version = "0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/seedlit/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 64a08fd43eea5c98cd7b7d4426f7646ad989e607ceb104047348b75d2aa2584b
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv "
+
+requirements:
+  build:
+    - git
+    - python
+    - pip
+    - setuptools
+  host:
+    - pip
+    - python >=3.6
```

You can combine these together and just use `host`. Also, I don't think you're actually using `git` in the build process since you're building from a tarball:
```suggestion
  host:
    - pip
    - setuptools
    - python >=3.6
```

---

I think that is a good point. I had read the docs, but was a bit confused with the exact usage of `build` and `host`. I will trust you on this.